lib/wallaby/browser.ex

```diff
@@ -1040,6 +1040,105 @@ defmodule Wallaby.Browser do
     current_url(session) == "about:blank"
   end
 
+  @doc """
+  Accepts all subsequent JavaScript dialogs in the given session.
+  """
+  def accept_dialogs(%Session{}=session) do
+    Driver.accept_dialogs(session)
+    session
+  end
+
+  @doc """
+  Dismisses all subsequent JavaScript dialogs in the given session.
+  """
+  def dismiss_dialogs(%Session{}=session) do
+    Driver.dismiss_dialogs(session)
+    session
+  end
+
+  @doc """
+  Accepts one alert dialog, which must be triggered within the specified `fun`.
+  Returns the message that was presented to the user. For example:
+
+  ```
+  message = accept_alert session, fn(s) ->
+    click(s, Query.link("Trigger alert"))
+  end
+  ```
+  """
+  def accept_alert(%Session{}=session, fun) do
+    Driver.accept_alert(session, fun)
+  end
+
+  @doc """
+  Accepts one confirmation dialog, which must be triggered within the specified
+  `fun`. Returns the message that was presented to the user. For example:
+
+  ```
+  message = accept_confirm session, fn(s) ->
+    click(s, Query.link("Trigger confirm"))
+  end
+  ```
+  """
+  def accept_confirm(%Session{}=session, fun) do
+    Driver.accept_confirm(session, fun)
+  end
+
+  @doc """
+  Dismisses one confirmation dialog, which must be triggered within the
+  specified `fun`. Returns the message that was presented to the user. For
+  example:
+
+  ```
+  message = dismiss_confirm session, fn(s) ->
+    click(s, Query.link("Trigger confirm"))
+  end
+  ```
+  """
+  def dismiss_confirm(%Session{}=session, fun) do
+    Driver.dismiss_confirm(session, fun)
+  end
+
+  @doc """
+  Accepts one prompt, which must be triggered within the specified `fun`. The
+  `[with: value]` option allows to simulate user input for the prompt. If no
+  value is provided, the default value that was passed to `window.prompt` will
+  be used instead. Returns the message that was presented to the user. For
+  example:
+
+  ```
+  message = accept_prompt session, fn(s) ->
+    click(s, Query.link("Trigger prompt"))
+  end
+  ```
+
+  Example providing user input:
+
+  ```
+  message = accept_prompt session, [with: "User input"], fn(s) ->
+    click(s, Query.link("Trigger prompt"))
+  end
+  ```
+  """
+  def accept_prompt(%Session{}=session, options \\ [], fun) do
+    input_value = Keyword.get(options, :with)
+    Driver.accept_prompt(session, input_value, fun)
+  end
```

Comment:
- [ ] Maybe we could add a safeguard here in case the supplied `input_value` is not a string, because  `window.prompt` will always return a string except when it's dismissed. For example, providing an integer as argument could be considered an error. Likewise, passing `nil` would be equivalent to calling `dismiss_prompt`, so that function should be used instead.

@PragTob @keathley what do you think? Should we raise an error here? Or maybe a warning would be more appropriate?